ui/src/data/containers/DetailPageContainer/DetailPageContainer.tsx

```diff
@@ -55,74 +57,102 @@ const DetailPage = ({
-                />
-              ) : (
-                <></>
-              )
-            }
+      <Row
+        className="__DetailPage__"
+        justify="center"
+        data-testid="data-detail-page-container"
+      >
+        <Col xs={24} md={22} lg={21} xxl={18}>
+          <Row className="mv3" justify="center" />
+          <Row
+            className="mv2"
+            justify="center"
+            gutter={{ xs: 0, lg: 16, xl: 32 }}
           >
-            <Row>
-              <Col>
-                <h2>
-                  <LiteratureTitle title={title} />
-                </h2>
-                <div>
-                  <AuthorsAndCollaborations
-                    authorCount={authorCount}
-                    authors={authors}
-                    collaborations={collaborations}
-                    enableAuthorsShowAll
-                    page="Data detail"
-                  />
+            <Col xs={24} lg={24}>
+              <ContentBox
+                className="sm-pb3"
+                leftActions={
+                  <>
+                    {urls && (
+                      <UrlsAction
+                        urls={urls}
+                        text="links"
+                        page="Data detail"
+                        trackerEventId="Data website"
+                      />
+                    )}
+                    {isSuperUserLoggedIn && (
+                      <APIButton url={window.location.href} />
+                    )}
+                  </>
+                }
+                rightActions={
+                  citationCount !== null && citationCount !== undefined ? (
+                    <IncomingLiteratureReferencesLinkAction
+                      itemCount={citationCount}
+                      referenceType="citation"
+                      linkQuery={getReferencingPapersQueryString(recordId)}
+                      trackerEventId="Citations link"
+                      eventCategory="Data search"
+                    />
+                  ) : (
+                    <></>
+                  )
+                }
+              >
+                <Row>
+                  <Col>
+                    <h2>
+                      <LiteratureTitle title={title} />
+                    </h2>
+                    <div>
+                      <AuthorsAndCollaborations
+                        authorCount={authorCount}
+                        authors={authors}
+                        collaborations={collaborations}
+                        enableAuthorsShowAll
+                        page="Data detail"
+                      />
+                    </div>
+                    {date && <LiteratureDate date={date} />}
+                  </Col>
+                </Row>
+                <div className="mt3">
+                  <DOIListShowAll dois={dois} />
+                  <ExperimentList experiments={acceleratorExperiments} />
+                  {literatureRecords && (
```

Comment:
Could you also add this in the test data and check how it looks ? so as to have a clear picture with all the data present